lib/features/down_by_maintenance/data/services/fb_check_server_status_service.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/entities/check_result.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/services/check_server_status_service.dart';
+
+/// Implementation of [ICheckServerStatusService] that uses Firebase Remote Config to check server status.
+class FirebaseCheckServerStatusService implements ICheckServerStatusService {
+  @override
+  final ValueNotifier<ServerCheckResult> serverStatus = ValueNotifier(ServerCheckResult.processing);
```

Comment:
It is better to replace it with `BehaviorSubject`.

Reply:
\+